api/src/controllers/login.js

```diff
@@ -496,6 +500,10 @@ module.exports = {
       })
       .catch(next);
   },
+  getSSOLogin: (req, res) => {
+    // Placeholder url. Change to redirect to KeyCloak UI for SSO login
+    res.redirect('https://www.google.com');
```

Comment:
update to `{base_url}/medic/login/oidc`

Reply:
you can ignore this comment. this function will not be necessary because #9765 adds the odic login path to the route and handles the redirection to oidc provider.